package/test_requirements.txt

```
@@ -6,5 +6,6 @@ pytest-coverage
 behave>=1.2.6, <2.0
 pylint>=2.3.1, <2.4.0 # 2.4.1 doesn't work for Python 3.5, and requires investigation.
 pytest-mock>=1.7.1,<2.0
+requests-mock>=1.6.0, <2.0.0
```

To be consistent, maybe

```suggestion
requests-mock>=1.6.0, <2.0
```

I can see `pytest-mock>=1.7.1,<2.0` and `behave>=1.2.6, <2.0` for example.